README.md

```diff
@@ -8,15 +8,24 @@ The Tableau Migration SDK is a client library to build an application to facilit
 
 ## Get started
 
-First, clone the repo:
-
-- Open a command line interface
-- `cd` to the parent directory where the repo directory will live
-- `git clone https://github.com/tableau/tableau-migration-sdk.git`
-
-After cloning the repo:
-
-- Open `Migration SDK.sln` using Visual Studio
+Quickstart:
+- For Python install using PIP `pip install tableau-migration`
+- For C# install using NuGet 
+   - [dotnet CLI](https://learn.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-using-the-dotnet-cli): `dotnet add package Tableau.Migration --version 1.0.0`
+   - [Nuget Package Manager](https://learn.microsoft.com/en-us/nuget/quickstart/install-and-use-a-package-in-visual-studio): Search for Tableau.Migration
+
+To look at sourcecode and delve deeper:
+  First, clone the repo:
+  
+  - Open a command line interface
+  - `cd` to the parent directory where the repo directory will live
+  - `git clone https://github.com/tableau/tableau-migration-sdk.git`
+  
+  After cloning the repo:
+  
+  - Open `Migration SDK.sln` using Visual Studio
+
+    Not using Visual Studio? Check out our [code samples](https://tableau.github.io/migration-sdk/samples/intro.html)
```

Comment:
This has nothing to do with "Not using visual studio".
These are just the samples, everyone should be looking at those.